src/renderer/src/components/AccountList.tsx

```diff
@@ -1,34 +1,24 @@
 import { Fragment, useEffect, useMemo, useState } from 'react'
-import { useTranslation } from 'react-i18next'
-import { TbMenu } from 'react-icons/tb'
-import { DndContext, DragEndEvent } from '@dnd-kit/core'
-import { SortableContext, useSortable, verticalListSortingStrategy } from '@dnd-kit/sortable'
```

Comment:
Remove the @dnd-kit/core and @dnd-kit/sortable package from the project, we are not using them anywhere else